cpp/include/cudf/search.hpp

```diff
@@ -125,34 +125,35 @@ std::unique_ptr<column> upper_bound(table_view const& t,
  * @param mr       Device memory resource to use for device memory allocation
  *
  * @return bool    If `value` is found in `column` true, else false.
- *---------------------------------------------------------------------------**/
+ */
 bool contains(column_view const& col, scalar const& value,
               rmm::mr::device_memory_resource* mr = rmm::mr::get_default_resource());
 
-/**---------------------------------------------------------------------------*
- * @brief Identify if elements in the column are contained in elements of a second column.
- *
- * For each element of col, identify if that element is contained in in_col
+/**
+ * @brief  Returns a new column of type bool8 identifying for each element of @p haystack column,
```

Comment:
Too long for a brief. Put a blank line after the first sentence to make it the brief.

Reply:
OK